README.rst

```diff
@@ -23,12 +23,11 @@ How to port the CSS from djangoproject.com
 Assumes that `code.djangoproject.com` and `djangoproject.com` are stored in the
 same directory (adjust paths if needed).
 
-    # Copy the generated CSS:
-    cp ../djangoproject.com/static/css/*.css trac-env/htdocs/css/
-    # Copy _utils.scss (needed by trackahacks.scss):
-    cp ../djangoproject.com/static/scss/_utils.scss trac-env/htdocs/css/
-    # Copy the javascript directory:
-    cp -rT ../djangoproject.com/static/js trac-env/htdocs/js
-    # Compile trackhacs.scss:
-    compass compile --sass-dir trac-env/htdocs/css/ --css-dir trac-env/htdocs/css/ -e development --sourcemap --force
-
+1. Copy the generated CSS:
+   ``cp ../djangoproject.com/static/css/*.css trac-env/htdocs/css/``
+2. Copy _utils.scss (needed by trackahacks.scss):
+   ``cp ../djangoproject.com/static/scss/_utils.scss trac-env/htdocs/css/``
+3. Copy the javascript directory:
+   ``cp -rT ../djangoproject.com/static/js trac-env/htdocs/js``
+4. Compile trackhacs.scss:
+   ``compass compile --sass-dir trac-env/htdocs/css/ --css-dir trac-env/htdocs/css/ -e development --sourcemap --force``
```

Comment:
Can you change this command to use `sassc` instead (I missed that when I added the list above).